CRM/Event/Form/SelfSvcTransfer.php

```diff
@@ -500,17 +498,15 @@ public function sendCancellation() {
   }
 
   /**
-   *  Move Participant registration to new contact.
+   * Move Participant registration to new contact.
    *
    * @param int $toContactID
-   * @param int $fromParticipantID
-   * @param int $eventID
    *
    * @throws \CRM_Core_Exception
    * @throws \CiviCRM_API3_Exception
    */
-  public function transferParticipantRegistration($toContactID, $fromParticipantID, $eventID) {
-    $query = 'select role_id, source, fee_level, is_test, is_pay_later, fee_amount, discount_id, fee_currency,campaign_id, discount_amount from civicrm_participant where id = ' . $fromParticipantID;
+  protected function transferParticipantRegistration($toContactID) {
```

Comment:
So  in terms of where we are going with this code - I'm hoping it would be moved to the BAO at some point as a move() function & called from the api. I mention this because this change makes the function more rather than less tied to the form it is on - which is the opposite direction

Reply:
cool. I've updated the PR to keep the fromParticipantID param :)